### PR TITLE
Update deck.js

### PR DESCRIPTION
**Pull request by KenzoIkeizume**
### Description

Including a prop types on deck.

Fixes # (issue)

#### Type of Change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

### How Has This Been Tested?

Pass a type in a deck component and view if breaks.

### CODE REVIEW

**Comment by kale-stew** [comment]
Thank you for updating these definitions!

I don't believe `contentWidth` or `contentHeight` is

1) set on the Deck, instead it's a propType of the Slide
2) is meant to be modified, it's more of an internal tracker for the component to render a magic-wrapper, fullscreen, or a mobile view.

If you remove these two types from the Deck, I would be happy to merge

File: src/components/deck.js [hunk 1]
```diff
@@ -28,7 +28,10 @@ export default class Deck extends Component {
     autoplayLoop: PropTypes.bool,
     autoplayOnStart: PropTypes.bool,
     children: PropTypes.node,
+    contentHeight: PropTypes.number,
```
Comment (kale-stew) [inline src/components/deck.js]:
```suggestion

```

File: src/components/deck.js [hunk 2]
```diff
@@ -28,7 +28,10 @@ export default class Deck extends Component {
...
+    contentWidth: PropTypes.number,
```
Comment (kale-stew) [inline src/components/deck.js]:
```suggestion
```

**Comment by KenzoIkeizume** [comment]
I added the suggestion, but I committed another. Could you view if this can be applied? Please :)

**Comment by kale-stew** [comment]
I do have to say, I'm not a fan of renaming `props` to `properties` in a single file, as it doesn't mesh with our conventions throughout the rest of the codebase, but it doesn't break anything! So to avoid more back and forth on this, I'm going to just merge. 👍

**Pull request merged**